packages/node_modules/pouchdb-collate/src/index.js

```diff
@@ -20,14 +20,11 @@ function collate(a, b) {
   if ((ai - bi) !== 0) {
     return ai - bi;
   }
-  if (a === null) {
-    return 0;
-  }
```

Comment:
this code path was only being hit if you did `collate(undefined, null)` and it turned out this was pointless after I fixed `lastKey` because we always coerce `undefined` to `null` everywhere else when emitted by map/reduce (as CouchDB itself does)